ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -178,8 +179,15 @@ ReadButtons.propTypes = {
   showReads: PropTypes.func,
 }
 
+const applyUserTrackSettings = (tracks, options) => tracks.map(track => ({
+  ...Object.keys(options).includes(track.type) ? { ...track, ...options[track.type] } : track,
```

Comment:
its more performant to just try and access the value directly, rather than convert the object to a key list and check if the list contains a string. So just `options[track.type] ? { ...track, ...options[track.type] } : track` and the same for the merged track below